core/model/__init__.py

```diff
@@ -584,3 +584,5 @@ def _bulk_operation(self):
 from .resource import Hyperlink, Representation, Resource, ResourceTransformation
 from .time_tracking import PlaytimeEntry, PlaytimeSummary
 from .work import Work, WorkGenre
+
+from core.lane import Lane, LaneGenre  # isort:skip
```

Comment:
Can you add a comment here about why the ordering is important? It'll save me from wondering why this isn't sorted and then learning the hard lessons of import hell all over again.

Reply:
Done!